src/displayapp/screens/WeatherSymbols.cpp

```diff
@@ -34,3 +34,28 @@ const char* Pinetime::Applications::Screens::Symbols::GetSymbol(const Pinetime::
       break;
   }
 }
+
+const char* Pinetime::Applications::Screens::Symbols::GetSimpleCondition(const Pinetime::Controllers::SimpleWeatherService::Icons icon) {
+  switch (icon) {
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Sun:
+      return "Clear";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudsSun:
+      return "Cloudy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Clouds:
+      return "Cloudy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::BrokenClouds:
+      return "Cloudy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudShowerHeavy:
+      return "Rainy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::CloudSunRain:
+      return "Rainy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Thunderstorm:
+      return "Stormy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Snow:
+      return "Snowy";
+    case Pinetime::Controllers::SimpleWeatherService::Icons::Smog:
+     return "Misty";
```

Comment:
I think these should follow the OWM main group names more closely:
- Clear
- Clouds
- Clouds
- Clouds
- Drizzle
- Rain
- Thunder(storm) <--- Thunderstorm doesn't fit, but Thunder does
- Snow
- Mist <--- This one doesn't have a single name for the main group, but I've seen several apps defaulting to Mist, since it's the first one in the atmosphere condition group.

![terminal-weather](https://github.com/InfiniTimeOrg/InfiniTime/assets/259780/d527ecc4-1043-40c4-b961-f433fe958ef8)